JamGame/Assets/Scripts/Level/Executor.cs

```diff
@@ -68,6 +74,17 @@ private IEnumerator DayStartRoutine(float time)
             ActionEndNotify?.Invoke();
         }
 
+        public void Execute(PreMeeting preMeeting)
+        {
+            if (location is IDataProvider<LocationImpl.AllEmployeesAtMeeting> dataProvider)
```

Comment:
You mustn't assume that location implements `IDataProvider<...>`, just link to it instead